src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -258,6 +268,7 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
           }),
       })
     } catch (error) {
+      errorRef.current = error
       logger.error("Error updating order", error)
       reject()
```

Comment:
non-blocking: I think this literally just came up for me while i was messing around in the TS playground on a daydream. the type of error in a catch is `any` (more specifically it is not a string, but react's useRef will treat it as `any` even though we know it is an `Error`): 
<img width="762" alt="image" src="https://github.com/user-attachments/assets/73f8604e-9240-4651-9a6d-2e9e517b80e1" />

What I think we could do here is:
- Attempt to get  `error.code` (from our exchange error types in the mp schema) or `error.message`
- check that against a list of known error codes from backends (making exchange error codes a graphql string enum could make this easier in the future)
- Always be able to fall back to the generic error.

Non blocking because if we don't have copy for every case today this could all be possible future work. In that case i would recommend something like 
```suggestion
      errorRef.current = error.message || "unknown_error"
      logger.error("Error updating order", error)
      reject()
```